editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -405,6 +456,19 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
     return nodeMetadata;
   }
 
+  function getAllowListAnswers(nodeId: string) {
+    const { data } = flow[nodeId];
+    const nodeFn = data?.fn || data?.val;
+    if (nodeFn && allowList.includes(nodeFn)) {
+      const answerIds = breadcrumbs[nodeId]?.answers;
```

Comment:
I think the reason this doesn't work going backwards is that the current approach has an assumption that we always want to update the analytics log for the previous node with the answer which is stored in the current breadcrumbs. 

When we move backwards through a flow the breadcrumbs are removed and the answers for the question go with them.

We could track what the saved answers were when a user loads a node they've already answered in the analytics logs. 

Although I'd be tempted not to and only track allow_list answers when a user explicitly hits continue. I think this might be more consistent with the broad approach to user's answers?